### PR TITLE
Thunderbird Fix for Mac OS X

### PR DESCRIPTION
**Pull request by derinb**
This commit fixes the toolbar button issue for Mac OS X Thunderbird

### CODE REVIEW

**Comment by jvillalobos** [comment]
This is easy enough to merge. Please address my comment and I'll accept it.

**Comment by derinb** [comment]
@jvillalobos what should I address for this PR please? Cant see your comment.

**Comment by jvillalobos** [comment]
(Sorry, new GitHub review UI)

File: extension/bootstrap.js [hunk 1]
```diff
@@ -174,12 +174,23 @@ var PersonasPlusBootstrapAddon = {
...
         // Get the list of browser windows already open
-        var windows = wm.getEnumerator("navigator:browser");
+        var windows = wm.getEnumerator(this.getWindowType());
         while (windows.hasMoreElements()) {
             var domWindow = windows.getNext().QueryInterface(Ci.nsIDOMWindow);
             WindowListener.tearDownBrowserUI(domWindow);
         }
     },
+    getWindowType: function() {
+        switch (Cc["@mozilla.org/xre/app-info;1"].getService(Ci.nsIXULAppInfo).ID) {
```
Comment (derinb) [inline extension/bootstrap.js]: Updated. Thanks.

**Pull request merged**